exercises/javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,10 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  var truth = false;
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      truth = true;
```

Comment:
This will not work if the `7` is not the last number. Try testing it with some other numbers following `7` in your array as well